### ISSUE TITLE
setuptools + shebang issue

### ISSUE THREAD

**Issue by anderkve** [jeriek/xsec#3]
[@jeriek and I discussed this on email -- just registering an issue here to log it properly. Jeriek already has a fix ready.]

There seems to be a problem with shebang expansion and setuptools. Here's how I can reproduce it with a clean Python2.7 environment:

```
virtualenv myenvs27/xsec_test
source myenvs27/xsec_test/bin/activate
pip install xsec==1.0.1
xsec-download-gprocs -t gg
```
This produces a `No such file or directory` error because it is looking for the Python binary in the wrong path. (Looks like setuptools has expanded the shebang in the source code.)

**Comment by jeriek** [comment]
Apparently setuptools changed the `#!/usr/bin/env python` shebangs in the scripts with the expanded local path on my machine when making the distribution files for PyPI. This seems to be an issue for others too (https://stackoverflow.com/questions/33792696/how-to-prevent-python-wheel-from-expanding-shebang). Running `python setup.py install sdist bdist_wheel` twice seems to resolve this for me, and I have tested on clean Python 2.7/3.6/3.7/3.8 environments on another machine. I will upload a new v1.0.2 release to PyPI later today.

**Comment by jeriek** [comment]
Resolved in [v1.0.2](https://github.com/jeriek/xsec/releases/tag/1.0.2).
